### ISSUE TITLE
Rule Request: LimitWildcardStaticImports

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#202]
We found that the Sonar rule java:S3030 - Classes should not have too many "static" imports - is not meeting our needs.
This makes sense:
```
With the default threshold value: 4

import static java.lang.Math.*;
import static java.util.Collections.*;
import static com.myco.corporate.Constants.*;
import static com.myco.division.Constants.*;
import static com.myco.department.Constants.*;  // Noncompliant
```
However, if static imports of individual members, limiting to 4 doesn't make sense:
```
import static com.myco.corporate.Constants.NUTS;
import static com.myco.corporate.Constants.CRAZY;
import static com.myco.corporate.Constants.FOOLISH;
import static com.myco.corporate.Constants.AWESOME;
import static com.myco.corporate.Constants.GREAT; // false positive, should be allowed
```
This is OK.

There is a rule java:S2208 - Wildcard imports should not be used - however, static imports are ignored by this rule.

So, we need a rule for limiting the number of wildcard static imports.
